Examples/Algorithms/TrackFinding/include/ActsExamples/TrackFinding/DefaultHoughFunctions.hpp

```diff
@@ -0,0 +1,150 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2020-2021 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
```

Comment:
Would it be better if these functions go to some namespace?

Reply:
Is there a recommendation for this? It's not clear if this should go into ActsExample? Otherwise?

Reply:
What about:
```
namespace ActsExamples {
  namespace DefaultHoughFunctions {
  }
}
```
A bit mouthful but will be clear where these functions came from.

Reply:
OK, I added that, fair